dirty.rb

```diff
@@ -0,0 +1,8 @@
+a,b=2,1
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
__[staging][rubocop]__
* Line is too long. [152/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)